libr/core/cbin.c

```diff
@@ -3888,6 +3923,9 @@ R_API int r_core_bin_info(RCore *core, int action, int mode, int va, RCoreBinFil
 	if ((action & R_CORE_BIN_ACC_SEGMENTS)) {
 		ret &= bin_sections (core, mode, loadaddr, va, at, name, chksum, true);
 	}
+	if ((action & R_CORE_BIN_ACC_SECTIONS_MAPPING)) {
+		ret &= rabin_map_sections_to_segments(core->bin);
```

Comment:
Are you willing to implement the json mode as well? As you can see, other actions in this function accept a `mode` argument, so they can print in different modes. There is no need, in this case, to have the other modes, but the json one is usually useful especially for r2pipe.

If not, it's ok anyway :)

Reply:
Nice, let me try.

Reply:
Added. Please review, thanks!